### PR TITLE
Keyboard updates: Polish, Danish UIS-6 and bug fixing

### LINKED ISSUE #1733: [Deutsch] The special characters (ä, ö or ü) are wrongly added in the search bar at long press and no other character can be added from the displayed list
**[Affected Versions]:**
- Firefox Reality 1.4-32450938 (RC2)

**[Affected Platforms]:**
- Oculus Go - Android 7.1.2
- Oculus Quest - Android 7.1.1
- HTC Vive Focus Plus - Android 7.1.2

**[Prerequisites]:**
- The headset is turned on and the browser is opened.
- The language of the on screen keyboard is set to German.

**[Steps to reproduce]:**
1. Click inside the awesomebar so that the keyboard appears.
2. Long press a special character (e.g. ä, ö, ü) and observe the behavior.
3. Click on a character from the list and observe the behavior.

**[Expected results]:**
Step 2: A special character list appears.
Step 3: A special character can be selected to be added in the input field.

**[Actual results]:**
Step 2: The long pressed special character is added in the input field.
Step 3: A list is displayed but the special character selected can not be added to the input field.

**[Notes]:**
- The long press functionality is working as expected for the other characters.
- Attached a screen recording of the issue: [here](https://drive.google.com/file/d/1zGLBffOoi7sJqKGzRlI1ZS68VJz9jHkS/view?usp=sharing).

### LINKED ISSUE #1750: Polish Keyboard
Implement Polish keyboard. Design is here:
https://trello.com/c/oqT3ElJz/439-de-69-polish-keyboard

### LINKED ISSUE #1751: Danish Keyboard
Implement Danish keyboard. Design is here:
https://trello.com/c/K0cfGN5a/438-de-68-danish-keyboard

### PR DESCRIPTION
**Pull request by keianhzo**
Fixes #1751 Fixes #1750 Fixes #1733 Added Danish and Polish and updated the Spec to UIS-6_Keyboard_021. Also fixes the long press bug.

The UIS-6_Keyboard_021 updated includes:
- Updating hover styles for popup and languages menu to azure
- Show the current language in the search bar for 3secs after it's selected
- Show disabled shift buttons in the symbols keyboard 

Also added Danish and Polish to the Display and Voice languages as part of #1594

### CODE REVIEW

File: app/src/main/res/values/non_L10n.xml [hunk 1]
```diff
@@ -79,6 +79,8 @@
     <string name="language_ru" translatable="false">Русский</string>
     <string name="language_ko_KR" translatable="false">한국어</string>
     <string name="language_it_IT" translatable="false">Italiano</string>
+    <string name="language_pl_PL" translatable="false">Polish</string>
+    <string name="language_da_DK" translatable="false">Danish</string>
```
Comment (bluemarvin) [inline app/src/main/res/values/non_L10n.xml]: These both need to be localized.

**Comment by keianhzo** [comment]
@bluemarvin This strings are displayed in the spacebar and have always only being displayed in the selected language. Should we change that?

**Comment by MortimerGoro** [comment]
@keianhzo I think is good to have them in their original language, that way to can choose the right language from any language selected. Or else you cannot understand any language if you mistakenly select an unknown language.

**Comment by jvonitter** [comment]
Yes, it would be good to have language names in the original/matching language

File: app/src/main/res/values/non_L10n.xml [hunk 2]
```diff
@@ -79,6 +79,8 @@
     <string name="language_ru" translatable="false">Русский</string>
     <string name="language_ko_KR" translatable="false">한국어</string>
     <string name="language_it_IT" translatable="false">Italiano</string>
+    <string name="language_pl_PL" translatable="false">Polskie</string>
```
Comment (bluemarvin) [inline app/src/main/res/values/non_L10n.xml]: I have it on good authority that `Polskie` should be `Polski`

**Comment by jvonitter** [comment]
related to #1964

**Comment by keianhzo** [comment]
@bluemarvin Updated Polski string
@jvonitter The spacebar language string is displayed in the native language. If you choose the Polish keyboard it's displayed as "Polski" no matter what display language is the app using. Is not still happening in this PR as we don't have the translations for those strings yet (so it falls back to the English string) but will happen when they are translated. I think is safe to close #1964

**Pull request merged**